bin/verilator

```diff
@@ -433,6 +433,7 @@ detailed descriptions of these arguments.
     --public-depth <level>      Mark public to specified module depth
     --public-params             Mark all parameters as public_flat
     --public-flat-rw            Mark all variables, etc as public_flat_rw
+    --public-off                Ignore all public comment markings, allowing for config files to exclusively control this
```

Comment:
I think `--public-off` as opposed to `--no-public` (which is supported as opposite of `--public`) is confusing.

Maybe `--public-ignore`?

Also make line shorter to match others.